src/panels/lovelace/cards/types.ts

```diff
@@ -118,85 +118,72 @@ export interface EnergyCardBaseConfig extends LovelaceCardConfig {
   collection_key?: string;
 }
 
-export interface EnergySummaryCardConfig extends LovelaceCardConfig {
+export interface EnergySummaryCardConfig extends EnergyCardBaseConfig {
   type: "energy-summary";
   title?: string;
-  collection_key?: string;
 }
 
-export interface EnergyDistributionCardConfig extends LovelaceCardConfig {
+export interface EnergyDistributionCardConfig extends EnergyCardBaseConfig {
   type: "energy-distribution";
   title?: string;
   link_dashboard?: boolean;
-  collection_key?: string;
 }
-export interface EnergyUsageGraphCardConfig extends LovelaceCardConfig {
+export interface EnergyUsageGraphCardConfig extends EnergyCardBaseConfig {
   type: "energy-usage-graph";
   title?: string;
-  collection_key?: string;
 }
 
-export interface EnergySolarGraphCardConfig extends LovelaceCardConfig {
+export interface EnergySolarGraphCardConfig extends EnergyCardBaseConfig {
   type: "energy-solar-graph";
   title?: string;
-  collection_key?: string;
 }
 
-export interface EnergyGasGraphCardConfig extends LovelaceCardConfig {
+export interface EnergyGasGraphCardConfig extends EnergyCardBaseConfig {
   type: "energy-gas-graph";
   title?: string;
-  collection_key?: string;
 }
 
-export interface EnergyWaterGraphCardConfig extends LovelaceCardConfig {
+export interface EnergyWaterGraphCardConfig extends EnergyCardBaseConfig {
   type: "energy-water-graph";
   title?: string;
-  collection_key?: string;
 }
 
-export interface EnergyDevicesGraphCardConfig extends LovelaceCardConfig {
+export interface EnergyDevicesGraphCardConfig extends EnergyCardBaseConfig {
   type: "energy-devices-graph";
   title?: string;
-  collection_key?: string;
   max_devices?: number;
 }
 
-export interface EnergyDevicesDetailGraphCardConfig extends LovelaceCardConfig {
+export interface EnergyDevicesDetailGraphCardConfig extends EnergyCardBaseConfig {
   type: "energy-devices-detail-graph";
   title?: string;
-  collection_key?: string;
   max_devices?: number;
 }
 
-export interface EnergySourcesTableCardConfig extends LovelaceCardConfig {
+export interface EnergySourcesTableCardConfig extends EnergyCardBaseConfig {
   type: "energy-sources-table";
   title?: string;
-  collection_key?: string;
 }
 
-export interface EnergySolarGaugeCardConfig extends LovelaceCardConfig {
+export interface EnergySolarGaugeCardConfig extends EnergyCardBaseConfig {
   type: "energy-solar-consumed-gauge";
   title?: string;
-  collection_key?: string;
 }
 
 export interface EnergySelfSufficiencyGaugeCardConfig
-  extends LovelaceCardConfig {
+  extends EnergyCardBaseConfig {
   type: "energy-self-sufficiency-gauge";
   title?: string;
-  collection_key?: string;
 }
 
-export interface EnergyGridGaugeCardConfig extends LovelaceCardConfig {
-  type: "energy-grid-result-gauge";
+export interface EnergyGridGaugeCardConfig extends EnergyCardBaseConfig {
```

Comment:
Can we rename this type too?

Reply:
Sorry it took me a while I didn't have the repo locally